satpy/etc/composites/abi.yaml

```diff
@@ -147,3 +147,345 @@ composites:
         - name: C13
     - name: C08
     standard_name: airmass
+
+###
+## Airmass using Channel14 RGB (own)
+  airmass_eum:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+      - name: C08
+      - name: C10
```

Comment:
It was pointed out to me that my other comment was wrong and that your tropical airmass RGB recipe is correct. I was actually talking about this `airmass_eum` (which for some reason I thought was the tropical one). @jhbravo any idea where this `airmass_eum` came from?

Reply:
that of `_eum`  I copied it from a couple of combinations that are in `enhancements/ahi.yaml`
and my intention was that you could make the **original combination of eumetsat** with the band 10.8 and also with the **advanced sensors ABI/AHI** with the band 10.3
I was really just curious to compare

Reply:
It could be in the case of the eumetsat recipe that the enhancement are doing the inversion.